storage_controller/src/service.rs

```diff
@@ -363,6 +363,15 @@ pub struct Config {
     /// assume it is running in a test environment and try to update neon_local.
     pub compute_hook_url: Option<String>,
```

Comment:
For external folks using our binaries (people in discord have asked about the hook before), I guess we will need to maintain this for some time & enable working with a pageserver hook but no safekeeper hook.  That's okay, we can clean it up eventually, just have to give folks in #neon-oss and #self-hosted some advance notice.

Reply:
Fair points, do you think we should also keep support of that field in `neon_local`?

Reply:
answer was no, but we should notify on discord.